datadog/import_datadog_logs_pipeline_test.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 const pipelineConfigForImport = `
-resource "datadog_logs_pipeline" "test_import" {
+resource "datadog_logs_customer_pipeline" "test_import" {
```

Comment:
Can you make that `custom` instead of `customer` everywhere ? Seems more appropriate.

Reply:
ok, will do.